src/odemis/driver/test/tescan_test.py

```diff
@@ -299,10 +483,12 @@ def test_move_abs(self):
         time.sleep(6)
         testing.assert_pos_almost_equal(self.stage.position.value, p)
 
+    @skip  # not working with the newer simulator Tescan Essence v1.2.2
```

Comment:
You can specify why you are skipping something:
```suggestion
    @skip("not working with the newer simulator Tescan Essence v1.2.2")
```